.cirrus.yml

```
@@ -29,7 +33,15 @@ build_task:
     populate_script: mkdir -p ${GRADLE_USER_HOME}/caches
   build_script:
     - source cirrus-env BUILD
-    - ./gradlew build
+    - function gradle(){ ./gradlew "$@"; }; export -f gradle
+    - gradle --version
+    - source set_gradle_build_version
+    - regular_gradle_build_deploy_analyze --exclude-task sonar
```

Once the analysis on Next is configure we can trigger it:
```suggestion
    - regular_gradle_build_deploy_analyze 
```

---

I will fix that in another PR when tackling SONARSCALA-4